backend/api/publishing.py

```
@@ -0,0 +1,989 @@
+        if field not in scenario_dict:
+            scenario_dict[field] = getattr(scenario, field, None)
+    # Fix learning_objectives if it's a string
+    if isinstance(scenario_dict.get('learning_objectives'), str):
+        items = [item.strip() for item in scenario_dict['learning_objectives'].split('\n') if item.strip()]
+        scenario_dict['learning_objectives'] = items
+    return scenario_dict
+
+@router.post("/{scenario_id}/clone")
+async def clone_scenario(
+    scenario_id: int,
+    db: Session = Depends(get_db)
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)